### PR TITLE
Key Metrics Setup CTA Dismissal

### PR DESCRIPTION
**Pull request by kuasha420**
## Summary

Addresses issue:

- #6232 

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [x] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [x] Ensure CI checks pass.
- [x] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [x] Ensure the PR has the correct target branch.
- [x] Double-check that the PR is okay to be merged.
- [x] Ensure the corresponding issue has a ZenHub release assigned.
- [x] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/components/KeyMetrics/KeyMetricsSetupCTAWidget.stories.js [hunk 1]
```diff
@@ -64,10 +70,35 @@ export default {
...
+					</div>
+					<div
+						style={ { flex: 1 } }
+						className={ classnames( {
+							// Turn off animations for non-mobile breakpoints. The standard VRT behaviour is to set
+							// animation-duration to 0ms, this does not play well with this component as there is a
+							// continual chain of animation at non-mobile breakpoints.
+							'googlesitekit-vrt-animation-none':
+								breakpoint !== BREAKPOINT_SMALL,
+						} ) }
```
Comment (techanvil) [inline assets/js/components/KeyMetrics/KeyMetricsSetupCTAWidget.stories.js:95]: This CSS class isn't needed, as there's no animation to disable for the `KeyMetricsSetupCTAWidget` component.

```suggestion
```

**Review by techanvil — CHANGES_REQUESTED** [review]
Hey @kuasha420, this is looking good, just a few minor comments from me to consider. Please take a look.

File: assets/js/components/KeyMetrics/KeyMetricsSetupCTAWidget.js [hunk 2]
```diff
@@ -69,10 +75,44 @@ function KeyMetricsSetupCTAWidget( { Widget, WidgetNull } ) {
 			select( MODULES_ANALYTICS_4 ).isGatheringData()
 	);
 
+	const showTooltip = useShowTooltip( KEY_METRICS_SETUP_CTA_WIDGET_SLUG );
+	const { isTooltipVisible } = useTooltipState(
+		KEY_METRICS_SETUP_CTA_WIDGET_SLUG
+	);
+	const isDismissed = useSelect( ( select ) =>
+		select( CORE_USER ).isItemDismissed( KEY_METRICS_SETUP_CTA_WIDGET_SLUG )
```
[the remark this answers is not in the document]
Comment (kuasha420) [reply assets/js/components/KeyMetrics/KeyMetricsSetupCTAWidget.js:83]: Hi, I have copied the implementation from the `AdBlockingRecoveryWidget` and not the `AdSenseSetupCTA`, that's why there's the difference. Both will most likely change to accommodate the change to support periodic dismissals, so I'd rather keep it consistent between this two. Cheers.

File: assets/js/components/KeyMetrics/KeyMetricsSetupCTAWidget.js [hunk 3]
```diff
@@ -69,10 +75,44 @@ function KeyMetricsSetupCTAWidget( { Widget, WidgetNull } ) {
 			select( MODULES_ANALYTICS_4 ).isGatheringData()
 	);
 
+	const showTooltip = useShowTooltip( KEY_METRICS_SETUP_CTA_WIDGET_SLUG );
+	const { isTooltipVisible } = useTooltipState(
+		KEY_METRICS_SETUP_CTA_WIDGET_SLUG
+	);
```
[the remark this answers is not in the document]
Comment (kuasha420) [reply assets/js/components/KeyMetrics/KeyMetricsSetupCTAWidget.js:81]: See my other comment.

File: assets/js/components/KeyMetrics/KeyMetricsSetupCTAWidget.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (techanvil) [reply assets/js/components/KeyMetrics/KeyMetricsSetupCTAWidget.js:83]: Ahh, thanks for the explanation. Makes sense, let's leave as-is then!

**Review by techanvil — APPROVED** [review]
LGTM, nice work @kuasha420!

**Pull request merged**